libc/config/linux/x86_64/entrypoints.txt

```diff
@@ -219,7 +219,7 @@ set(TARGET_LIBC_ENTRYPOINTS
     libc.src.sys.epoll.epoll_pwait
     # TODO: Need to check if pwait2 is available before providing.
     # https://github.com/llvm/llvm-project/issues/80060
-    # libc.src.sys.epoll.epoll_pwait2
+    libc.src.sys.epoll.epoll_pwait2
```

Comment:
have all of the buildbots been updated to new enough kernel versions to support pwait2?

Reply:
I can check; though I should drop this change.  Was using it just for local testing and enabling this entrypoint is orthogonal to the intent of this PR.

Reply:
dropped in https://github.com/llvm/llvm-project/pull/87017/commits/4e2833ab58e1d64c95cda19338370899f62d15fd